src/modules/enhancedPOI/docs/it_IT.md

```diff
@@ -0,0 +1,10 @@
+L'impostazione dei PDI è fondamentale per alcune missioni come `Incidente aereo di grandi dimensioni fuori aeroporto` per generarle. Se non desideri utilizzare i PDI di altre persone, dovrai impostare i tuoi. Questo modulo ti permette di impostare i PDI in modo più semplice.
+
+* Tutti i PDI del tipo selezionato sono evidenziati
+  * Il colore di evidenziazione può essere modificato nelle impostazioni.
+* Se il marcatore lascia l'area della mappa visibile, il marcatore viene riportato al centro della mappa.
+* Puoi mostrare e nascondere ogni tipo di POI
+  * Questo può essere controllato quando si apre il `Impostare un nuovo menu PDI`
+ 
+ 
+ ! [Esempio di barra](POI.png)
```

Comment:
Remove the space ` ` between `!` and `[`.

Reply:
Corrected